app/main.development.js

```diff
@@ -0,0 +1,338 @@
+import { app, BrowserWindow, Menu, shell, ipcMain } from 'electron';
+import open from 'open';
+import { autoUpdater } from 'electron-updater';
+import { sync as syncActions } from './actions';
+
+let menu;
+let template;
+let mainWindow = null;
+
+if (process.env.NODE_ENV === 'production') {
+  const sourceMapSupport = require('source-map-support'); // eslint-disable-line
+  sourceMapSupport.install();
+}
+
+if (process.env.NODE_ENV === 'development') {
+  require('electron-debug')(); // eslint-disable-line global-require
+  const path = require('path'); // eslint-disable-line
+  const p = path.join(__dirname, '..', 'app', 'node_modules'); // eslint-disable-line
+  require('module').globalPaths.push(p); // eslint-disable-line
+}
+
+app.on('window-all-closed', () => {
+  if (process.platform !== 'darwin') app.quit();
+});
+
+
+const installExtensions = async () => {
+  if (process.env.NODE_ENV === 'development') {
+    const installer = require('electron-devtools-installer'); // eslint-disable-line global-require
+
+    const extensions = [
+      'REACT_DEVELOPER_TOOLS',
+      'REDUX_DEVTOOLS'
+    ];
+
+    const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
+
+    // TODO: Use async interation statement.
+    //       Waiting on https://github.com/tc39/proposal-async-iteration
```

Comment:
This is another case of being a comment directly from the boilerplate. It's probably something to look into for the future, but may not be a v1 (async generators would be somewhat of a learning experience for me).